README.md

```diff
@@ -369,12 +433,12 @@ Note: This will also disable the Go Back button for sub menus.
 You can manually add exit and go back buttons using the following:
 
 ```php
-use PhpSchool\CliMenu\CliMenuBuilder;
+<?php
+
+use PhpSchool\CliMenu\Builder\CliMenuBuilder;
 use PhpSchool\CliMenu\Action\ExitAction;
 use \PhpSchool\CliMenu\Action\GoBackAction;
```

Comment:
Shouldn't we use the leading slash everytime ?
I guess it doesn't matter much for docs examples, then just remove it here for consistency.

Reply:
Actually its recommended against in the PHP docs, will remove.